src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -290,7 +290,7 @@ protected function componentComposer()
         }
 
         // Make sure not installed plugins are not present in composer.json
-        $installedPackages = $this->getJson('vendor/composer/installed.json');
+        $installedPackages = $this->getJson('vendor/composer/installed.json', throwException: false);
```

Comment:
Not sure if this syntax is correct, can you double check?

Reply:
Hi @joaocsilva good catch. 
  Actually, it was PhpStorm that suggested this, and I chose it blindly. 
Now when checked in documentation I see that, these are so called "named arguments" introduced in PHP 8.0.0. 
  In this particular case it's not needed to use it, since they are intended to skip over multiple optional parameters.

**Please let me know if you think it's better to remove.**

Reply:
Yes I would prefer to not use if it is not needed

Reply:
Hi @joaocsilva it's fixed.